semantic-conventions/src/tests/data/yaml/metrics.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: metric.foo
+    prefix: bar
+    type: metric_group
+    brief: "This document defines foo."
+    note: >
+        Details about foo.
+    attributes: 
+      - id: egg.type
+        type: string
+        brief: 'Type of egg.'
+        examples: ["chicken", "emu", "dragon"]
+
+  - id: metric.foo.size
+    prefix: foo
+    type: metric
+    extends: metric.foo
```

Comment:
Does the "extends" mechanism really make sense here? I guess it's fine as a prototype, but it will always copy all the attributes, and the brief/etc.

Reply:
Yeah, fair point. It wasn't necessary here. All tests still pass without it. I've removed the extends here, it was superfluous.